lib/browser/api-util.ts

```diff
@@ -13,4 +13,14 @@ Zone.__load_patch('util', (global: any, Zone: ZoneType, api: _ZonePrivate) => {
   api.patchMethod = patchMethod;
   api.bindArguments = bindArguments;
   api.patchMacroTask = patchMacroTask;
+  // load blackListEvents from global
+  const SYMBOL_BLACK_LISTED_EVENTS = Zone.__symbol__('BLACK_LISTED_EVENTS');
+  const SYMBOL_DISABLE_EVENTS = Zone.__symbol__('DISABLE_EVENTS');
```

Comment:
I'm not sure about this name.. I think it should be "DISABLED_EVENTS", or "UNPATCHED_EVENTS" or "UNINTEREPTED_EVENTS". 

because you'll use it declaratively rather than imperatively as in:

```
Zone.__symbol__('DISABLED_EVENTS') = ['scroll'];
```


"DISABLE_EVENTS" would be more suitable if the api looked like:

```
Zone.disableEvents("scroll", "mouseover", "somethingelse")
```

Reply:
@IgorMinar, thanks, sure, I believe `DISABLED_EVENTS` is better, I just copied it from https://github.com/angular/angular/pull/28529, I will change it and also add comment in the `angular` PR.

Reply:
I think `DISABLED_EVENTS` is good, but `UNPATCHED_EVENTS` might be even better because it doesn't imply that things like scrolling just don't work.

Reply:
@kara, thanks, you are right, I will change it to `UNPATCHED_EVENTS`.

Reply:
Yes! Unlatched is the best